src/syslog/syslog.c

```diff
@@ -64,3 +63,22 @@ void syslog_transport_close(syslog_transport_t *transport) {
     }
     transport->is_open = false;
 }
+void dns_found_cb(const char *name, const ip_addr_t *ipaddr, void *callback_arg) {
+    return;
+}
+
+bool syslog_transport_open(syslog_transport_t *transport, const char *host, int port) {
+    ip_addr_t addr;
+    err_t res = dns_gethostbyname(host, &addr, NULL, NULL);
+    static char ip_addres[16];
+    if (res == ERR_OK) {
+        // ip address already cached or the name was valid ip address
+        return syslog_transport_open_ip4(transport, addr, port);
+    }
+    if (res == ERR_INPROGRESS) {
+        // dns lookup queued, wait for it to finish
+        return false;
+    }
+    // failed return false
+    return false;
+}
```

Comment:
What would be -very- useful are logs for events like 
- DNS failed to resolve
- DNS resolved to X